contracts/nft/src/query/nft_info.rs

```diff
@@ -0,0 +1,24 @@
+use cosmwasm_std::{to_binary, Binary, Deps, Env, QuerierWrapper, StdError};
+use cw721::NftInfoResponse;
+use provwasm_std::types::provenance::metadata::v1::{MetadataQuerier, ScopeResponse};
+
+use crate::core::error::ContractError;
+use crate::core::msg::NftData;
+use crate::storage::nft::TOKENS;
+
+pub fn handle(deps: Deps, _env: Env, token_id: String) -> Result<Binary, ContractError> {
+    let info = TOKENS.load(deps.storage, &token_id)?;
+
+    Ok(to_binary(&NftInfoResponse {
+        token_uri: None,
```

Comment:
Same question about token_uri. Should this be None?

Reply:
yeah same thing as the other one